### DISCUSSION TITLE
the new opengl3 loader doesn't support glGetError?

### DISCUSSION THREAD

**Discussion by shi-yan** [ocornut/imgui#4508] [New Users Build/Link/Run issues ONLY!]
I saw that gl3w has been removed. My code is broken now, because there is no glGetError with the new opengl3_loader.h, what should I do?

**Comment by ocornut** [comment]
Your code _should_ use the standard OpenGL headers or the loader of your choice, why would you need to use our loader?

**Reply by ocornut** [reply]
(Sorry was a typo yesterday made the sentence hard to understand, edited)

**Comment by ghost** [comment]
I think that is because you have ImGui with your source code and not in a separate solution?  So your code adds the loader and ImGui adds it as well.  This is what happened to me.  I think ImGui is supposed to be in a different solution.  Unless I am understanding something incorrectly, which can be the case.

**Reply by PathogenDavid** [reply]
Dear ImGui should not need to be built separately from the rest of your app.

**Reply by ocornut** [reply]
Either way this doesn't make any difference.

**Comment by ocornut** [comment]
Your code your use the standard OpenGL headers, why would you need to use our loader?
